### PR TITLE
(PDB-2355) deprecated cli import/export

### PR DESCRIPTION
**Pull request by wkalt**
This deprecates the CLI import/export tools in favor of the db subcommand.

### CODE REVIEW

File: documentation/anonymization.markdown [hunk 1]
```diff
@@ -9,11 +9,28 @@ The export tool has the ability to anonymize the PuppetDB archive before returni
 Using the `export` command
 -----
 
-To create an anonymized PuppetDB archive directly, simply run the `puppetdb export` tool, preferably on your PuppetDB instance:
+To create an anonymized PuppetDB archive directly, use the Puppet `db` subcommand
+from any node with puppetlabs-client-tools installed:
```
Comment (wkalt) [inline documentation/anonymization.markdown]: this is a made-up package -- do we have a name yet?

Comment (ajroetker) [inline documentation/anonymization.markdown]: For PE it's pe-client-tools, @jdwelch for the FOSS version are we calling this puppet-client-tools?

**Comment by mullr** [comment]
@jdwelch can you comment on this?

File: documentation/anonymization.markdown (hunk 1 above)
Comment (ajroetker) [inline documentation/anonymization.markdown]: Looks like we're calling this `puppet-client-tools` @wkalt

**Comment by jdwelch** [comment]
Big +1 to "puppet-client-tools"

**Comment by ajroetker** [comment]
Fwiw I've renamed the package https://github.com/puppetlabs/ci-job-configs/pull/955 and https://github.com/puppetlabs/puppetdb-cli-vanagon/pull/2

**Comment by MosesMendoza** [comment]
Does this mean puppetdb CLI will now be included in the pe-client-tools package? 

The project https://github.com/puppetlabs/puppet-client-tools-vanagon builds pe-client-tools.

https://docs.puppetlabs.com/pe/latest/install_pe_client_tools.html#installing-the-pe-client-tools-package

**Comment by jdwelch** [comment]
@MosesMendoza Eventually, yes.

**Comment by ajroetker** [comment]
@MosesMendoza sorry this is just a confusion of naming. The PuppetDB CLI will only be a FOSS thing for PDB4, the package this project is in will be called `puppet-client-tools` built in the `puppetdb-cli-vanagon` repo. It is not going to be built yet (until RBAC support) in the `puppet-client-tools-vanagon` repo which builds the `pe-client-tools` package. Naming is shit.

**Comment by MosesMendoza** [comment]
So the current plan is to ship an open source package called 'puppet-client-tools' that contains the PDB CLI?

**Comment by ajroetker** [comment]
@MosesMendoza yep

**Comment by MosesMendoza** [comment]
I have some reservations about the naming - is there a JIRA issue with this decision in it that I can comment on?

**Comment by ajroetker** [comment]
There's an email thread, I'll add you to the thread @MosesMendoza

**Comment by MosesMendoza** [comment]
@ajroetker super sorry for delaying your work here - I'll write something up in the thread quickly

**Comment by ajroetker** [comment]
Looks like this needs a rebase now

**Comment by ajroetker** [comment]
@wkalt Also this should be a PR against stable right?

File: documentation/anonymization.markdown [hunk 2]
```diff
@@ -9,11 +9,28 @@ The export tool has the ability to anonymize the PuppetDB archive before returni
 Using the `export` command
 -----
 
-To create an anonymized PuppetDB archive directly, simply run the `puppetdb export` tool, preferably on your PuppetDB instance:
+To create an anonymized PuppetDB archive directly, use the Puppet `db` subcommand
+from any node with puppet-client-tools installed:
```
Comment (mullr) [inline documentation/anonymization.markdown]: This is a ghost comment

**Comment by kbarber** [comment]
@wkalt this needs a rebase.

**Pull request merged**